src/seedsigner/gui/components.py

```diff
@@ -610,7 +610,8 @@ def __post_init__(self):
         self.accent_font = Fonts.get_font(GUIConstants.FIXED_WIDTH_EMPHASIS_FONT_NAME, self.font_size)
 
         # Fixed width font means we only have to measure one max-height character
-        char_width, char_height = self.font.getsize("Q")
+        (left, top, right, bottom) = self.font.getbbox("Q")
+        char_width, char_height = right - left, bottom - top
```

Comment:
Minor note, but I would split the `char_width` and `char_height` assignments into their own lines. Reading tuple-ized math is weird.